enterprise/frontend/src/embedding-sdk/components/public/StaticDashboard/StaticDashboard.tsx

```diff
@@ -37,7 +38,7 @@ const _StaticDashboard = ({
         theme: userTheme,
         font,
         hideDownloadButton,
-        hideParameters,
+        hideParameters: hideParameters.join(","),
```

Comment:
This is a deal breaker. you gotta provide the value otherwise it will crash. I tested that you cannot omit `hideParameters`

Reply:
Also, I'd try to ensure we ship one kind of work in a single PR. This PR actually contains 2
1. fix(sdk): Improve external API. (This makes `hideParameters` easier to use by users)
2. doc(sdk): Add static embed doc.

Although, I'd say it's fine for now to just have a single PR for this.

Reply:
Yes. Please let's fix this.